build.gradle

```diff
@@ -69,11 +69,11 @@ dependencies {
     compile ("com.pahimar.ee3:EquivalentExchange3:${ee3_version}:dev") {
         transitive = false
     }
-    
```

Comment:
They can be undone (i.e. I can undo them for you), but keeping to an aggressive trailing white space removal policy can really help prevent future merge conflicts.